proto/ibc/transfer/transfer.proto

```diff
@@ -65,8 +65,12 @@ message DenomTrace {
 // NOTE: To prevent a single token from being transferred, set the TransfersEnabled parameter to
 // true and then set the bank module's SendEnabled parameter for the denomination to false.
```

Comment:
this godoc should be updated, also see my previous [comment](https://github.com/cosmos/cosmos-sdk/pull/6607#discussion_r472087812)